android/cli/lib/base-builder.js

```diff
@@ -54,6 +54,16 @@ AndroidBaseBuilder.prototype.writeXmlFile = function writeXmlFile(srcOrDoc, dest
 		}
 		fs.writeFileSync(dest, '<?xml version="1.0" encoding="UTF-8"?>\n' + srcDoc.toString());
 		return;
+	} else {
```

Comment:
Generally I prefer to put the truthy block first. It makes the code easier to read.